### PR TITLE
add pandamic

### PR DESCRIPTION
**Pull request by swiftrax**
## Description
Add pandamic 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by swiftrax** [comment]
image
PID provided by MICROCHIP

File: keyboards/handwired/swiftrax/pandamic/config.h [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x04D8
+#define PRODUCT_ID      0xEB0E
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Swiftrax
+#define PRODUCT         Pandamic
+#define DESCRIPTION     Southpaw 65%
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/config.h:28]:
```suggestion
```

File: keyboards/handwired/swiftrax/pandamic/info.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
...:7.75, "y":1}, {"x":8.75, "y":1}, {"x":9.75, "y":1}, {"x":10.75, "y":1}, {"x":11.75, "y":1}, {"x":12.75, "y":1}, {"x":13.75, "y":1}, {"x":14.75, "y":1}, {"x":15.75, "y":1}, {"x":16.75, "y":1}, {"x":17.75, "y":1, "w":1.5}, {"x":19.75, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":4.25, "y":2, "w":1.75}, {"x":6, "y":2}, {"x":7, "y":2}, {"x":8, "y":2}, {"x":9, "y":2}, {"x":10, "y":2}, {"x":11, "y":2}, {"x":12, "y":2}, {"x":13, "y":2}, {"x":14, "y":2}, {"x":15, "y":2}, {"x":16, "y":2}, {"x":17, "y":2, "w":2.25}, {"x":19.75, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3, "h":2}, {"x":4.25, "y":3, "w":1.25}, {"x":5.5, "y":3}, {"x":6.5, "y":3}, {"x":7.5, "y":3}, {"x":8.5, "y":3}, {"x":9.5, "y":3}, {"x":10.5, "y":3}, {"x":11.5, "y":3}, {"x":12.5, "y":3}, {"x":13.5, "y":3}, {"x":14.5, "y":3}, {"x":15.5, "y":3}, {"x":16.5, "y":3, "w":1.75}, {"x":19.75, "y":3}, {"x":18.5, "y":3.25}, {"x":0, "y":4, "w":2}, {"x":2, "y":4}, {"x":4.25, "y":4, "w":1.25}, {"x":5.5, "y":4, "w":1.25}, {"x":6.75, "y":4, "w":1.25}, {"x":8, "y":4, "w":6.25}, {"x":14.25, "y":4}, {"x":15.25, "y":4}, {"x":16.25, "y":4}, {"x":17.5, "y":4.25}, {"x":18.5, "y":4.25}, {"x":19.5, "y":4.25}]
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/info.json:9]: Got a slight sequence error here.

```suggestion
            "layout": [{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4.25, "y":0}, {"x":5.25, "y":0}, {"x":6.25, "y":0}, {"x":7.25, "y":0}, {"x":8.25, "y":0}, {"x":9.25, "y":0}, {"x":10.25, "y":0}, {"x":11.25, "y":0}, {"x":12.25, "y":0}, {"x":13.25, "y":0}, {"x":14.25, "y":0}, {"x":15.25, "y":0}, {"x":16.25, "y":0}, {"x":17.25, "y":0}, {"x":18.25, "y":0}, {"x":19.75, "y":0}, {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1, "h":2}, {"x":4.25, "y":1, "w":1.5}, {"x":5.75, "y":1}, {"x":6.75, "y":1}, {"x":7.75, "y":1}, {"x":8.75, "y":1}, {"x":9.75, "y":1}, {"x":10.75, "y":1}, {"x":11.75, "y":1}, {"x":12.75, "y":1}, {"x":13.75, "y":1}, {"x":14.75, "y":1}, {"x":15.75, "y":1}, {"x":16.75, "y":1}, {"x":17.75, "y":1, "w":1.5}, {"x":19.75, "y":1}, {"x":0, "y":2}, {"x":1, "y":2}, {"x":2, "y":2}, {"x":4.25, "y":2, "w":1.75}, {"x":6, "y":2}, {"x":7, "y":2}, {"x":8, "y":2}, {"x":9, "y":2}, {"x":10, "y":2}, {"x":11, "y":2}, {"x":12, "y":2}, {"x":13, "y":2}, {"x":14, "y":2}, {"x":15, "y":2}, {"x":16, "y":2}, {"x":17, "y":2, "w":2.25}, {"x":19.75, "y":2}, {"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3, "h":2}, {"x":4.25, "y":3, "w":1.25}, {"x":5.5, "y":3}, {"x":6.5, "y":3}, {"x":7.5, "y":3}, {"x":8.5, "y":3}, {"x":9.5, "y":3}, {"x":10.5, "y":3}, {"x":11.5, "y":3}, {"x":12.5, "y":3}, {"x":13.5, "y":3}, {"x":14.5, "y":3}, {"x":15.5, "y":3}, {"x":16.5, "y":3, "w":1.75}, {"x":18.5, "y":3.25}, {"x":19.75, "y":3}, {"x":0, "y":4, "w":2}, {"x":2, "y":4}, {"x":4.25, "y":4, "w":1.25}, {"x":5.5, "y":4, "w":1.25}, {"x":6.75, "y":4, "w":1.25}, {"x":8, "y":4, "w":6.25}, {"x":14.25, "y":4}, {"x":15.25, "y":4}, {"x":16.25, "y":4}, {"x":17.5, "y":4.25}, {"x":18.5, "y":4.25}, {"x":19.5, "y":4.25}]
```

File: keyboards/handwired/swiftrax/pandamic/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/keymaps/default/keymap.c:1]: Please add a license header.

File: keyboards/handwired/swiftrax/pandamic/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "", 
+    "url": "", 
+    "maintainer": "qmk",
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/info.json:4]:
```suggestion
    "keyboard_name": "Pandamic",
    "url": "https://github.com/swiftrax",
    "maintainer": "swiftrax",
```

File: keyboards/handwired/swiftrax/pandamic/keymaps/info.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/keymaps/info.json:1]: This file should be deleted.

File: keyboards/handwired/swiftrax/pandamic/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/keymaps/via/keymap.c:1]: Please add a license header.

File: keyboards/handwired/swiftrax/pandamic/pandamic.c [hunk 7]
```diff
@@ -0,0 +1 @@
+#include "pandamic.h"
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/pandamic.c:1]: Please add a license header.

File: keyboards/handwired/swiftrax/pandamic/pandamic.h [hunk 8]
```diff
@@ -0,0 +1,25 @@
+#pragma once
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/pandamic.h:1]: Please add a license header.

File: keyboards/handwired/swiftrax/pandamic/readme.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Pandamic
+
+A southpaw 65% mechanical keyboard with rotary encoder support
+
+* Keyboard Maintainer: Swiftrax
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/readme.md:5]:
```suggestion
* Keyboard Maintainer: [Swiftrax](https://github.com/swiftrax)
```

File: keyboards/handwired/swiftrax/pandamic/readme.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+* Hardware Supported: Pandamic
+* Hardware Availability: https://github.com/swiftrax
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/readme.md:7]:
```suggestion
* Hardware Availability: [GitHub.com](https://github.com/swiftrax/Pandamic)
```

File: keyboards/handwired/swiftrax/pandamic/rules.mk [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/rules.mk:11]:
```suggestion
```

File: keyboards/handwired/swiftrax/pandamic/rules.mk [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/rules.mk:28]:
```suggestion
```

File: keyboards/handwired/swiftrax/pandamic/rules.mk [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```
Comment (noroadsleft) [inline keyboards/handwired/swiftrax/pandamic/rules.mk:31]:
```suggestion
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/10227/files).

File: keyboards/handwired/swiftrax/pandamic/keymaps/info.json (hunk 5 above)
Comment (swiftrax) [reply keyboards/handwired/swiftrax/pandamic/keymaps/info.json:1]: so do I delete this info.json?

Comment (noroadsleft) [reply keyboards/handwired/swiftrax/pandamic/keymaps/info.json:1]: This one. `info.json` doesn't do anything at keymap-level, only at keyboard/revision-level.

**Review by noroadsleft — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Comment by swiftrax** [comment]
Thanks noradsleft and fauxpark!

**Pull request merged**